src/radiation/radiation.cpp

```diff
@@ -112,6 +112,16 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   bool absorption = pin->GetOrAddBoolean("radiation", "absorption", true);
   params.Add("absorption", absorption);
 
+  if (method == "cooling_function") {
+    const bool do_liebendorfer =
+        pin->GetOrAddBoolean("radiation", "do_liebendorfer", false);
+    const bool do_lightbulb = pin->GetOrAddBoolean("radiation", "do_lightbulb", false);
+    const Real lum = pin->GetOrAddReal("radiation", "lum", 4.0e52);
+    params.Add("radiation", "do_liebendorfer");
+    params.Add("radiation", "do_lightbulb");
+    params.Add("radiation", "lum");
```

Comment:
These lines are causing the tests to fail. Syntax is:
```suggestion
    params.Add("do_liebendorfer", do_liebendorfer);
    params.Add("do_lightbulb", do_lightbulb);
    params.Add("lum", lum);
```
`params` is already owned by the `radiation` package.